### PR TITLE
Chinese localization supplement and fixes

### PR DESCRIPTION
**Pull request by snnh**

### CODE REVIEW

File: src/locales/zh_CN/growth.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-    "Medium_Fast": "较快",
-    "Medium_Slow": "较慢",
-    "Slow": "慢",
-    "Fluctuating": "最慢"
+    "Erratic": "Erratic",
+    "Fast": "Fast",
+    "Medium_Fast": "Medium Fast",
+    "Medium_Slow": "Medium Slow",
+    "Slow": "Slow",
+    "Fluctuating": "Fluctuating"
```
Comment (TempsRay) [inline src/locales/zh_CN/growth.ts:9]: This is English now?

File: src/locales/zh_CN/starter-select-ui-handler.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ export const starterSelectUiHandler: SimpleTranslationEntries = {
     "cycleVariant": 'V: 切换变种',
     "enablePassive": "启用被动",
     "disablePassive": "禁用被动",
-    "locked": "未解锁",
-    "disabled": "已禁用",
+    "locked": "Locked",
+    "disabled": "Disabled",
```
Comment (TempsRay) [inline src/locales/zh_CN/starter-select-ui-handler.ts:42]: This file and all of modifier-type.ts seem to have been reverted back to english?

**Review by TempsRay — CHANGES_REQUESTED** [review]
Why is there changes included that go from Chinese back to English?

**Comment by Somod1** [comment]
something wrong here, some file back to English

**Comment by bennybroseph** [comment]
Closing this for the moment due to the above issues. Once this is resolved, feel free to reopen it

**Pull request closed without merging**